backend/sqlite/sqlite.go

```diff
@@ -403,6 +403,9 @@ func (be *sqliteBackend) CreateOrchestrationInstance(ctx context.Context, e *bac
 
 	var instanceID string
 	if err := be.createOrchestrationInstanceInternal(ctx, e, tx, &instanceID); err != nil {
+		if err == backend.ErrSkipExistInstance {
```

Comment:
In Go, it's preferred to do `errors.Is(err, backend.ErrSkipExistinstance)` instead of `err == backend.ErrSkipExistInstance`. More info here: https://go.dev/blog/go1.13-errors.

```suggestion
		if errors.Is(err, backend.ErrSkipExistinstance) {
```